### ISSUE TITLE
I want to ask, why didn't you use regularization loss on discriminative loss?

### ISSUE THREAD

**Issue by maftuhm** [harryhan618/LaneNet#18]
I saw in the second paper it was said that there are three components to discriminatory loss: variance, distance, and regulization loss. and you define reg_loss in your code. but you did not use it because you said it was not used in the original paper. So the loss function you use is the loss from the first paper and the loss from the second paper, exclude reg_loss. why?

loss function you used:
loss = a * seg_loss + b * var_loss + c * dist_loss

why not:
loss = a * seg_loss + b * var_loss + c * dist_loss + d * reg_loss

**Comment by harryhan618** [comment]
Hello @maftuhm , If you read carefully in the LaneNet paper, you would find that reg_loss is not used.

**Comment by akshatbhutra** [comment]
@harryhan618 can you please tell me how can I print the summary of layers of LaneNet Architecture (using tensorflow and keras). Thanks.
